examples/ido-pool/programs/ido-pool/src/lib.rs

```diff
@@ -18,8 +18,7 @@ pub mod ido_pool {
         end_deposits_ts: i64,
         end_ido_ts: i64,
     ) -> Result<()> {
-        if !(ctx.accounts.clock.unix_timestamp < start_ido_ts
```

Comment:
If `clock` isn't needed anymore, can you remove it from the `InitializePool` accounts struct and client.